src/Authentication/Model/AuthenticationEvent.cs

```diff
@@ -23,7 +23,7 @@ public class AuthenticationEvent
         /// <summary>
         /// Date, time of the authentication event. Set by producer of logevents
         /// </summary>
-        public DateTime Created { get; set; }
+        public DateTimeOffset Created { get; set; }
```

Comment:
This should probably be a `DateTimeOffset?`. Otherwise you have no good way to detect if it wasn't set.